### ISSUE TITLE
Simplify macOS setup

### ISSUE THREAD

**Issue by SubJunk** [UniversalMediaServer/UniversalMediaServer#1781]
At the moment we have a disk image or a wizard installer, but something I've seen programs doing which is cool, is the program checks whether it's being run from the applications directory and if not, offers to move itself. If we can do that we would have the best of all worlds

**Comment by esabol** [comment]
Not everyone wants every application to be in /Applications/. On some Macs, I intentionally install some applications in $HOME/Applications/ instead, for example.

What you really want to do is to detect if the application is being run **from the disk image** and then offer to copy it to /Applications. Just my two cents....

**Comment by SubJunk** [comment]
@esabol I agree and have edited the issue to match what you said

**Comment by insherahdwikatt** [comment]
Hi! I'm a beginner in open source and I would love to work on this issue.  
Is it still open? I'd be happy to take it and start contributing!  
I’m still learning, but I’ll do my best and follow your contributing guidelines!

**Comment by esabol** [comment]
> Is it still open?

Yes, nobody has implemented this feature yet. It would make a nice contribution. Best wishes on your open source journey!

**Comment by SubJunk** [comment]
That would be great, thanks!

**Comment by ibrakhim2906** [comment]
@SubJunk Hello! I would like to work on this feature? Do contribution still relevant?

**Comment by SubJunk** [comment]
Go for it
